integration/layer_reuse_test.go

```diff
@@ -25,6 +25,9 @@ func testRebuildLayerReuse(t *testing.T, context spec.G, it spec.S) {
 
 		name   string
 		source string
+
+		pullPolicy = "never"
+		// extenderBuildStr = ""
```

Comment:
Was this comment out intentional?  Same below

Reply:
removed!